src/components/form/form/form.tsx

```diff
@@ -115,9 +115,15 @@ export const Form: React.FC<FormProps> = ({
   const _nonFieldErrors = forceArray(nonFieldErrors);
 
   const [valuesState, setValuesState] = useState(initialValues);
-  useEffect(() => values && setValuesState(values), [values]);
   const [errorsState, setErrorsState] = useState(errors || {});
-  useEffect(() => errors && setErrorsState(errors), [errors]);
+
+  useEffect(() => {
+    return values && setValuesState(values);
+  }, [values]);
+
+  useEffect(() => {
+    return errors && setErrorsState(errors);
```

Comment:
I assume the reason for the `return` here is for the cleanup function? I don't believe this is necessary as the cleanup function is intended to clean up side effects (e.g. event listeners). This could just be a simple 
`values && setValuesState(values);` or with an extended `if (..) {...}`

Reply:
Nice catch, this is me not paying attentions, will fix!